ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/qos-profiles.j2

```diff
@@ -61,6 +61,47 @@ QOS Profile: **{{ profile.name }}**
 {%                 endfor %}
 {%             endif %}
 {%         endif %}
+
```

Comment:
So _above_ the blank line, you want to insert some logic that parse the queues looking for ecn key. Only if you find any, you include the table.

Reply:
Add logic here to check if queues have any ecn thresholds defined. If there are none found, the generation of the table is skipped.